server/utils/workspace/workspace_utils.py

```
@@ -12,34 +12,39 @@
 logger = logging.getLogger(__name__)
 
 
-def get_or_create_workspace(user_id: str, workspace_name: str = "default") -> Dict[str, Any]:
+def get_or_create_workspace(user_id: str, workspace_name: str = "default", org_id: str = None) -> Dict[str, Any]:
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)